app/models/listing.rb

```diff
@@ -16,6 +16,8 @@ class Listing < ApplicationRecord
   has_many :matches_as_receiver, class_name: "Match", foreign_key: "receiver_id"
   has_many :matches_as_provider, class_name: "Match", foreign_key: "provider_id"
 
+  accepts_nested_attributes_for :location
```

Comment:
i know we've gone back and forth. 

last i understood, we were only going to put Location on Person 
(and ServiceArea, Organization, and CommunityResources (if different than Organization's Location))

open to this approach too. maybe we just go w it since it's written that way here?

Reply:
cool. i say let's add it now, and if we remove later, great. much harder to add data after the fact than to trim it down! :)

Reply:
having it on listing will help w running queries to plan dispatcher drops, so i like it this way too...